projects/storefrontapp-e2e-cypress/cypress/integration/regression/asm/asm.e2e-spec.ts

```diff
@@ -72,4 +84,41 @@ context('Assisted Service Module', () => {
     // TODO(#9445): Add e2e test for this scenario
     it.skip('agent logout when user was logged and emulated should restore the session', () => {});
   });
+
+  describe('Apparel Site', () => {
+    before(() => {
+      Cypress.env('BASE_SITE', APPAREL_BASESITE);
+    });
+
+    after(() => {
+      Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
+    });
+
+    // This test only works if "sap-commerce-cloud-user-id" is added to the allowed headers of "corsfilter.commercewebservices.allowedHeaders" on the Commerce Cloud side
+    it.skip("should fetch products in a category based on the emulated user's authentication", () => {
```

Comment:
We could update the CI backends to accept this header and enable the tests.
If we decide to go for a feature level flag, we might not have a choice but to update the CI backends before we merge this PR.  Let's discuss.